### PR TITLE
Windows dev section in the contributing file

### PR DESCRIPTION
**Pull request by jplu**
# What does this PR do?

This PR adds a section for people who wants to contribute from a Windows environment.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -308,3 +308,14 @@ Check our [documentation writing guide](https://github.com/huggingface/transform
 for more information.
 
 #### This guide was heavily inspired by the awesome [scikit-learn guide to contributing](https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md)
+
+
+### Develop on Windows
+
+The Windows developers will need some more tools to properly run and test transformers. Here a minimum list of the tools that have to be installed:
+
+1. Install [Miniconda for Windows](https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe)
```
Comment (sgugger) [inline CONTRIBUTING.md:317]: Why miniconda specifically? There are plenty of other ways to use Python on Windows.

Comment (jplu) [reply CONTRIBUTING.md:317]: There are, but here I give at least one.

Comment (jplu) [reply CONTRIBUTING.md:317]: There are also plenty of other ways to use/install make.

Comment (sgugger) [reply CONTRIBUTING.md:317]: I don't see why we would recommend one way to install Python for Windows when we don't for other OS. It's also not in the right place: you can't really install transformers without python. I'd leave the section to "Here is one way you can run the make commands"

**Review by sgugger — APPROVED** [review]
Thanks!

**Review by LysandreJik — APPROVED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -308,3 +308,12 @@ Check our [documentation writing guide](https://github.com/huggingface/transform
...
 #### This guide was heavily inspired by the awesome [scikit-learn guide to contributing](https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md)
+
+
+### Develop on Windows
+
+One way one can run the make command on Window is to pass by MSYS2:
+
+1. [Download MSYS2](https://www.msys2.org/), we assume to have it installed in C:\msys64
+2. Open the command line C:\msys64\msys2.exe (it should be available from the start menu)
+3. Run in the shell: `pacman -Syu` and install make with `pacman -S make`
```
Comment (LysandreJik) [inline CONTRIBUTING.md:319]: pacman under windows! That's meta!

**Review by LysandreJik — APPROVED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (jplu) [reply CONTRIBUTING.md:319]: So true ahah

**Pull request merged**